SimCalorimetry/HGCalSimProducers/plugins/HGCDigitizer.cc

```diff
@@ -201,25 +200,25 @@ namespace {
         int waferThickness = getCellThickness(geom, detectorId);
         float cell_threshold = tdcForToAOnset[waferThickness - 1];
         const auto& hitRec = hitmapIterator.second;
-        float accChargeForToA(0.f), fireTDC(0.f);
+        float fireTDC(0.f);
         const auto aboveThrPos = std::upper_bound(
             hitRec.begin(), hitRec.end(), std::make_pair(cell_threshold, 0.f), [](const auto& i, const auto& j) {
               return i.first < j.first;
             });
 
         if (aboveThrPos == hitRec.end()) {
-          accChargeForToA = hitRec.back().first;
-          fireTDC = 0.f;
+          //          fireTDC = 0.f;
```

Comment:
Would it make the code more readable if the 2 conditions are combined?

Reply:
> Would it make the code more readable if the 2 conditions are combined?

Why not....
It is up to you, anyhow. I did not want to modify too much the structure of the code, but if you want I can implement it here as well.

Reply:
👍🏻

Reply:
@rovere done